cirq-web/cirq_ts/e2e/bloch_sphere/test_bloch_sphere.ts

```diff
@@ -0,0 +1,61 @@
+
+import puppeteer, { Browser } from 'puppeteer';
+import {expect} from 'chai';
+import {readFileSync} from 'fs';
+import pixelmatch from 'pixelmatch';
+import * as PNG from 'pngjs';
+
+
+/**
+ * Generates an HTML script with the current repository bundle
+ * that we will use to compare.
+ */
+
+// Due to the path, reading the file will only work by running "npm run start"
+const bundle_string = readFileSync('dist/bloch_sphere.bundle.js');
+const browserContent = `
+<!doctype html>
+<html lang="en">
+    <head>
+    <title>Cirq Web Development page</title>
+    </head>
+    <body>
+    <div id="container"></div>
+    <script>${bundle_string}</script>
+    <script>
+        CirqTS.showSphere('{"radius": 5}');
+    </script>
+    </body>
+</html>
+`;
+
+
+/**
+ * Testing to see if they look the same.
+ * WRITE ACTUAL TO A TEMP FOLDER
+ */
+describe('Check Bloch Sphere looks correct', () => {
+  //Opens a headless browser with the generated HTML file and takes a screenshot.
+  let browser: Browser;
+
+  before(async () => {
+    browser = await puppeteer.launch();
+    const page = await browser.newPage();
+    await page.setContent(browserContent);
+    await page.screenshot({path: 'e2e/bloch_sphere/bloch_sphere.png'});   
+    });
+
+  after(async () => {
+    await browser.close();
+  })
+  
+  it('Bloch sphere with |0⟩ statevector is correct', () => {
+    const expected = PNG.PNG.sync.read(readFileSync('e2e/bloch_sphere/bloch_sphere_expected.png'))
+    const actual = PNG.PNG.sync.read(readFileSync('e2e/bloch_sphere/bloch_sphere.png'))
```

Comment:
We should write this conditioned on an environment variable to either a temporary directory, or to the `bloch_sphere_expected.png`

Reply:
I have it in a temp directory, but wdym by conditioned on an environment variable ? Ie, user sets whether or not they want to have a file generated, and based on that setting, it will build it in a temp or not? Included in the commit that fixes this, I have a `utils/` folder that allows the user to generate a current screenshot of the bloch sphere via the current bundle.